### ISSUE TITLE
t/format.t test fails with latvian locale: "empty format produces default format"

### ISSUE THREAD

**Issue by richlv** [barefootcoder/date-easy#9]
installing Date-Easy-0.02 with 'cpan'.
fails at :

```
t/format.t ................ 1/?
#   Failed test 'empty format produces default format'
#   at t/format.t line 33.
#                   'S , 03 feb 2001 04:05:06 UTC'
#     doesn't match '(?-xism:^\w+, 03 \w+ 2001 04:05:06 UTC$)'
# Looks like you failed 1 test of 6.
```

RHEL6, perl v5.10.1

this only happens if locale is "lv_LV.UTF-8" - setting it to "C" passes the test

**Comment by barefootcoder** [comment]
Not sure how this ticket fell through the cracks.

From the output there, I'd say your particular locale produces a space in the day-of-the-week name.  It's unconventional, but I can easily change my regex to allow it.  (Although possibly I should just override `$LC_TIME` or something along those lines to try to guarantee myself a format.  But I'm not sure that doesn't make the test somewhat useless, so I'll avoid that for now.)

Fix for this should go out with the next version of Date::Easy.  Thx for reporting it.

**Comment by barefootcoder** [comment]
Eek.  It's actually a non-breaking space.  I think I'll just allow any character up to the first comma.  (Mainly I'm leaving this note for myself so I remember exactly what was going on.  But either way it's fixed now, in the code at least.)

**Comment by richlv** [comment]
Thank you for picking this up and the fix :)

**Comment by barefootcoder** [comment]
I've just released 0.09 containing this fix, so I'm closing the issue.
